packages/clay-form/src/Toggle.tsx

```diff
@@ -45,7 +45,7 @@ const ClayToggle = React.forwardRef<HTMLLabelElement, IToggleProps>(
 		}
 
 		return (
-			<span
+			<label
 				className={classNames(
 					'toggle-switch',
 					'simple-toggle-switch',
```

Comment:
Some users might not want to use `simple-toggle-switch`. I've seen commerce use toggle switches in narrow columns that make `simple-toggle-switch` look funny. Do we have a way to only use `toggle-switch`?